warehouse/internal/repo/schema.go

```diff
@@ -250,6 +415,8 @@ func (sh *WHSchema) GetTablesForConnection(ctx context.Context, connections []wa
 			SELECT max(id) FROM ` + whSchemaTableName + `
 			WHERE
 				(source_id, destination_id) IN (` + strings.Join(sourceIDDestinationIDPairs, ", ") + `)
+			AND
```

Comment:
Don't we also need a `enableTableLevelSchema` implementation here ?

Reply:
Planning to add it later, once we completely migrated. https://linear.app/rudderstack/issue/WAR-859/cleanup-for-table-level-schema-stop-populating-schema-and-no

Reply:
lets add a todo in this pr

Reply:
No sure if adding TODO is the correct things, since we are not sure about complete migration to table level schemas as of now.

Reply:
Not sure when we will be completely migrating it. Since we are already facing issues with schema TTL. Adding this will complicate it.